src/main/java/org/owasp/validator/html/scan/AntiSamyDOMScanner.java

```diff
@@ -577,11 +575,7 @@ private boolean processAttributes(Element ele, String tagName, Tag tag, int curr
                             currentAttributeIndex--;
                             addError(ErrorMessageUtil.ERROR_ATTRIBUTE_INVALID,
                               new Object[]{tagName, HTMLEntityEncoder.htmlEntityEncode(name), HTMLEntityEncoder.htmlEntityEncode(value)});
-
-                            if ("removeTag".equals(onInvalidAction) || "filterTag".equals(onInvalidAction)) {
-                                return true;
-                                // remove/filter the tag
-                            }
+                            
```

Comment:
@spassarop - For this one, this if / return was removed entirely by @LiuXing-R. Is that correct here? The default is to return false after the loops complete (on line ~601).

Reply:
It's ok too.